src/ir/comp.rs

```diff
@@ -357,8 +359,8 @@ impl CompInfo {
 
         let has_destructor = self.has_destructor ||
                              match self.kind {
-            CompKind::Union => false,
-            CompKind::Struct => {
+                                 CompKind::Union => false,
```

Comment:
I don't think these changes are really acceptable in general.

Is this only due to a newer version of rustfmt?